code/__DEFINES/turfs.dm

```diff
@@ -27,6 +27,9 @@
 
 #define TURF_FROM_COORDS_LIST(List) (locate(List[1], List[2], List[3]))
 
+/// Returns a list of turfs in the rectangle specified by the two coordinates
+#define BLOCK_COORDS(tx1, ty1, tz1, tx2, ty2, tz2) block(locate(tx1, ty1, tz1), locate(tx2, ty2, tz2))
```

Comment:
may I suggest
```suggestion
#define CORNER_BLOCK(corner, width, height) block(locate(corner.x, corner.y, corner.z), locate(min(corner.x + width, world.maxx), min(corner.y + height, world.maxy), corner.z))
```
or something like it
it's better for the case you're working with, and avoids overgeneralizing stuff that doesn't necessarily apply
could even do like, `BLOCK_OUTLINE(bottom_left, top_right) (hand back 1 smaller/larger on each edge)` if that'd be any better